code/game/objects/items/blueprints.dm

```diff
@@ -19,12 +19,10 @@
 	var/const/ROOM_ERR_SPACE = -1
 	var/const/ROOM_ERR_TOOLARGE = -2
 
-/obj/item/blueprints/attack_self(mob/M as mob)
-	if (!istype(M,/mob/living/carbon/human))
```

Comment:
I've contacted alberyk about this. He told me that it's the old code and it completely fine to do this. Kinda his part of code though.